api/serializers.py

```diff
@@ -2061,7 +2061,7 @@ class SearchEmergencySerializer(serializers.Serializer):
 class SearchSurgeAlertSerializer(serializers.Serializer):
     id = serializers.IntegerField()
     name = serializers.CharField()
-    keywords = serializers.ListField(serializers.CharField())
+    keywords = serializers.ListField()
```

Comment:
Hey @szabozoltan69, Any reason why we are removing child definition here?
Maybe we need to use child=serializers.CharField() instead?

Reply:
Placed back (and the other places also) as
```
...  = serializers.ListField(child=serializers.CharField())
```